libraries/DataFlash/examples/DataFlash_AllTypes/make.inc

```diff
@@ -0,0 +1,8 @@
+LIBRARIES += AP_Common # for new() overrides
+LIBRARIES += AP_Math
+LIBRARIES += AP_Mission
+LIBRARIES += AP_Param
+LIBRARIES += DataFlash
+LIBRARIES += GCS_MAVLink
+LIBRARIES += StorageManager
+LIBRARIES += AP_Scheduler
```

Comment:
This is missing waf integration, which is much simpler than the make one. Could you add it?